### PR TITLE
Copy airflow python spec to GCS

### PR DESCRIPTION
**Pull request by rajivpb**
The bucket name that Composer expects is hardcoded for now; will be fixed as part of https://github.com/googledatalab/pydatalab/issues/501

This PR just has the helper routines for this.

### CODE REVIEW

File: google/datalab/contrib/pipeline/_pipeline.py [hunk 1]
```diff
@@ -84,8 +69,35 @@ def get_airflow_spec(self):
...
+    bucket = client.get_bucket('airflow-staging-test36490808-bucket')
+    filename = 'dags/{0}.py'.format(self._name)
+    blob = storage.blob.Blob(filename, bucket)
+    blob.upload_from_string(self._get_airflow_spec())
+
+  @staticmethod
+  def get_pipeline_spec(spec_str, env=None):
+    """ Gets a dict representation of the pipeline-spec, given a yaml string.
+    Args:
+      name: name of the pipeline (line argument) from Datalab
```
Comment (qimingj) [inline google/datalab/contrib/pipeline/_pipeline.py]: arg name mismatch?

Comment (rajivpb) [reply google/datalab/contrib/pipeline/_pipeline.py]: Fixed.

File: tests/pipeline/pipeline_tests.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ class PipelineTest(unittest.TestCase):
 
   @staticmethod
   def _create_context():
-    project_id = 'test'
+    project_id = 'test.project'
```
Comment (jimmc) [inline tests/pipeline/pipeline_tests.py]: Read project IDs can not contain a period, only lowercase letters, digits, or hyphens.

Comment (rajivpb) [reply tests/pipeline/pipeline_tests.py]: Removed; thanks!

**Pull request merged**